src/style.css

```
@@ -1 +1,15 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.groups {
+  margin-bottom: 20px;
+  padding-bottom: 0;
```

```suggestion
  padding-bottom: 0;
```
checklist p.9 Remember, any other styles besides margins, should be browser default.